frontend/src/components/maps/leaflet/InfoSlideOut/InfoContent.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+import { ListGroup } from 'react-bootstrap';
+import { IBuilding, IParcel, PropertyTypes } from 'actions/parcelsActions';
+import { Label } from 'components/common/Label';
+import { ParcelPIDPIN } from './ParcelPIDPIN';
+import ParcelAttributes from './ParcelAttributes';
+import './InfoSlideOut.scss';
+import useKeycloakWrapper from 'hooks/useKeycloakWrapper';
+import Claims from 'constants/claims';
+
+export type IInfoContent = {
+  /** the selected property information */
+  propertyInfo: IParcel | IBuilding | null;
+  /** The property type [Parcel, Building] */
+  propertyTypeId: PropertyTypes | null;
+};
+
+/**
+ * Component that displays the appropriate information about the selected property
+ * in the property info slideout
+ * @param propertyInfo the selected property
+ * @param propertyTypeId the property type [Parcel, Building]
+ */
+export const InfoContent: React.FC<IInfoContent> = ({ propertyInfo, propertyTypeId }) => {
+  const keycloak = useKeycloakWrapper();
+  return (
+    <>
+      <ListGroup>
+        <Label className="header">Parcel Identification</Label>
+        {propertyTypeId === PropertyTypes.PARCEL && (
+          <ParcelPIDPIN parcelInfo={propertyInfo as IParcel} />
+        )}
+        <ListGroup.Item>
+          <Label>Name:</Label>
+          {propertyInfo?.name}
+        </ListGroup.Item>
+        <ListGroup.Item>
+          <Label>Owning Agency:</Label>
+          {propertyInfo?.agency}
+        </ListGroup.Item>
+        {propertyInfo?.subAgency && (
+          <ListGroup.Item>
+            <Label>Sub-agency:</Label>
+            {propertyInfo.subAgency}
+          </ListGroup.Item>
+        )}
+        <ListGroup.Item>
+          <Label>Classification:</Label>
+          {propertyInfo?.classification}
+        </ListGroup.Item>
+      </ListGroup>
+      <ListGroup>
+        <Label className="header">Location data</Label>
+        <ListGroup.Item>
+          <Label>Civic Address:</Label>
+          {propertyInfo?.address}
+        </ListGroup.Item>
+        <ListGroup.Item>
+          <Label>Location:</Label>
+          {propertyInfo?.administrativeArea}
+        </ListGroup.Item>
+        <ListGroup.Item>
+          <Label>Latitude:</Label>
+          {propertyInfo?.latitude}
+        </ListGroup.Item>
+        <ListGroup.Item>
+          <Label>Longitude:</Label>
+          {propertyInfo?.longitude}
+        </ListGroup.Item>
+      </ListGroup>
+      {(keycloak.hasAgency(propertyInfo?.agencyId as number) ||
```

Comment:
I wasn't sure what would be viewable to owning vs non-owning agencies so I put the check here

Reply:
We'll want an answer before merging this. Any input @bbjornholt ?

Reply:
@agalbrai There is a separate mockup for the non-owning agencies indicating the fields they should be able to see. I'll send you a link.